micro-api/controller/dataapps/sqleditor/sqleditor_api.go

```diff
@@ -0,0 +1,242 @@
+/******************************************************************************
+ * Copyright (c)  2023 PingCAP, Inc.                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");            *
+ * you may not use this file except in compliance with the License.           *
+ * You may obtain a copy of the License at                                    *
+ *                                                                            *
+ * http://www.apache.org/licenses/LICENSE-2.0                                 *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ ******************************************************************************/
+
+package sqleditor
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/pingcap/tiunimanager/common/client"
+	"github.com/pingcap/tiunimanager/message/dataapps/sqleditor"
+	"github.com/pingcap/tiunimanager/micro-api/controller"
+)
+
+// CreateSession
+// @Summary create session
+// @Schemes
+// @Description create session result
+// @Tags sqleditor
+// @Param cluster_id path string true "cluster ID"
+// @Param request_body body sqleditor.CreateSessionReq true "request_body"
+// @Accept json
+// @Produce json
+// @Success 200 {object} sqleditor.CreateSessionRes
+// @Router /{cluster_id}/session [post]
```

Comment:
maybe the @Router should start with '/dataapps/sqleditor/' ? Like '/dataapps/sqleditor/{cluster_id}/session'.
Same for the rest routers.